synthtool/gcp/templates/python_library/docs/summary_overview.md

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Note, this file may need to be excluded in `owlbot.py` manually downstream for repositories that publish to googleapis.dev instead of c.g.c since they won't have the corresponding `summary_class.html`, `summary_method.html` and `summary_property.html`

Reply:
After a closer look, we could exclude them in code by using `self.excludes += ["docs/summary_overview.md"]` when `is_google_cloud_api` is `False`.

Add an else block here for the split repo
https://github.com/googleapis/synthtool/blob/349943b54f15f9f9655a3d1c35d58acb68165678/synthtool/gcp/common.py#L299-L301

and here for the monorepo
https://github.com/googleapis/synthtool/blob/349943b54f15f9f9655a3d1c35d58acb68165678/synthtool/gcp/common.py#L235-L237

```
        # If the directory `google/cloud` exists, add kwargs to signal that the client library is for a Cloud API
        if Path("google/cloud").exists():
            kwargs["is_google_cloud_api"] = True
        else:
            self.excludes += ["docs/summary_overview.md"]
```

Reply:
Done.